gmplot/gmplot.py

```diff
@@ -165,6 +166,42 @@ def _process_heatmap_kwargs(self, settings_dict):
 
         return settings_string
 
+    def groundoverlay(self, url, bounds_dict):
```

Comment:
Since the google docs call the "Ground Overlays" can we switch to "ground_overlay" instead of "groundoverlay"?